src/angularComponent/ngcOmniboxController.js

```diff
@@ -1,5 +1,7 @@
 import {KEY, isSelectKey, isVerticalMovementKey} from '../coreComponent/keyboard.js';
 
+/* global setTimeout, clearTimeout */
```

Comment:
Why do you need this comment? Is there some environment configuration you need to do in .eslintrc?